tests/test_iqm_sampler.py

```diff
@@ -127,6 +128,13 @@ def test_credentials_are_passed_to_client():
     assert sampler._client._credentials.password == user_auth_args['password']
 
 
+@pytest.mark.usefixtures('unstub')
+def test_client_signature_is_passed_to_client():
+    """Test that IQMSampler set client signature"""
+    sampler = IQMSampler('http://some-url.iqm.fi', Adonis())
+    assert sampler._client._signature == f'iqm-client {version("iqm-client")}, cirq-iqm {version("cirq-iqm")}'
```

Comment:
Same comment as for qiskit, no reason to test `IQMClient::_signature` format:
```suggestion
    assert f'cirq-iqm {version("cirq-iqm")}' in sampler._client._signature
```

Reply:
fixed